pages/services/nifi/0.4.0-1.8.0/troubleshooting/debugging/index.md

```diff
@@ -0,0 +1,200 @@
+
+The following command should be used to get a list of available pods. In this example we are querying a service named {{ model.serviceName }}-dev.
+
+  ```shell
+   dcos {{ model.serviceName }} --name={{ model.serviceName }}-dev pod list
+  ```
+The following command should then be used to replace the pod residing on the failed machine, using the appropriate pod_name provided in the above list.
+
+  ```shell
+  dcos {{ model.serviceName }} --name={{ model.serviceName }}-dev pod replace <pod_name>
+  ```
+The pod recovery may then be monitored via the recovery plan.
+
+  ```shell
+  dcos {{ model.serviceName }} --name={{ model.serviceName }}-dev plan show recovery
+  ```
+
+# Restarting a Node
+
+If you must forcibly restart a pod’s processes but do not wish to clear that pod’s data, use the following command to restart the pod on the same agent machine where it currently resides. This will not result in an outage or loss of data.
+
+The following command should be used to get a list of available pods. In this example we are querying a service named {{ model.serviceName }}-dev.
+
+  ```shell
+  dcos {{ model.serviceName }} --name={{ model.serviceName }}-dev pod list
+  ```
+
+The following command should then be used to restart the pod, using the appropriate pod_name provided in the above list.
+
+  ```shell
+  dcos {{ model.serviceName }} --name={{ model.serviceName }}-dev pod restart <pod_name>
+  ```
+
+The pod recovery may then be monitored via the recovery plan.
+
+  ```shell
+  dcos {{ model.serviceName }} --name={{ model.serviceName }}-dev plan show recovery
+  ```
+
+# Accidentially deleted Marathon task but not service
+
+A common user mistake is to remove the scheduler task from Marathon, which doesn’t do anything to uninstall the service tasks themselves. If you do this, you have two options:
+
+## Uninstall the rest of the service
+
+If you really wanted to uninstall the service, you just need to complete the normal package uninstall steps described under  Uninstall.
+
+## Recover the Scheduler
+
+If you want to bring the scheduler back, you can do a dcos package install using the options that you had configured before. This will re-install a new scheduler that should match the previous one (assuming you got your options right), and it will resume where it left off. To ensure that you don’t forget the options your services are configured with, we recommend keeping a copy of your service’s options.json in source control so that you can easily recover it later.
+
+# ‘Framework has been removed’
+
+Long story short, you forgot to run janitor.py the last time you ran the service. See Uninstall for steps on doing that. In case you’re curious, here’s what happened:
+
+    1. You ran dcos package {{ model.serviceName }} --app-id {{ model.serviceName }}. This destroyed the scheduler and its associated tasks, but didn’t clean up its reserved resources.
```

Comment:
Do not indent a bulleted list. This is showing up as a code block.

Reply:
will take care from next time